### PR TITLE
Fix not able to accept multiple files types

### PR DESCRIPTION
**Pull request by unimonkiez**
**What kind of change does this PR introduce?**
- [X] Bug Fix
- [ ] Feature
- [ ] Refactoring
- [ ] Style
- [ ] Build
- [ ] Chore
- [ ] Documentation
- [ ] CI

**Did you add tests for your changes?**
- [X] Yes, my code is well tested
- [ ] Not relevant

**If relevant, did you update the documentation?**
- [ ] Yes, I've updated the documentation
- [X] Not relevant

**Summary**
Can't accept more than one file type, tried to accept both images and videos and ran into a problem.

**Reproduce**
[Demo on website](https://react-dropzone.js.org/#section-accepting-specific-file-types) - Can't pick png, only first option is available to pick.

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by paulhklam1122** [comment]
Thank you so much @unimonkiez! @rolandjitsu If we could please get this merged, that would be greatly appreciated! Been wanting to support both JPGs and PNGs for an image file upload tool.

**Comment by rolandjitsu** [comment]
@unimonkiez you cannot see al file types because there's an option to pick.

Screenshot 2022-06-11 at 10 03 11

**Comment by taylor-cedar** [comment]
@rolandjitsu That options thing is very bad UX. We can't expect users to know or understand that if developers can't find that. This happens in OSX and Ubuntu. Please can we merge this PR to make all file type work by default?

**Comment by rolandjitsu** [comment]
@taylor-cedar I suggest you submit that feedback to the Apple and Ubuntu teams. We're using the options as [showOpenFilePicker](https://developer.mozilla.org/en-US/docs/Web/API/window/showOpenFilePicker).

**Comment by taylor-cedar** [comment]
@rolandjitsu Did you take a look at this PR? The change in this PR makes the code work as expected. No need contact Apple or Ubuntu. This library is using the API incorrectly for [showOpenFilePicker](https://developer.mozilla.org/en-US/docs/Web/API/window/showOpenFilePicker) to make both file types work without going into the options. There is no bug with Apple or Ubuntu. It's behaving as expected.

**Comment by unimonkiez** [comment]
> @unimonkiez you cannot see al file types because there's an option to pick.
> 
> Screenshot 2022-06-11 at 10 03 11

Wow totally missed that, the PR makes it work without using the picker at the bottom though

**Comment by pabacham** [comment]
Could we escalate this PR to be merged asap ?

File: src/utils/index.js [hunk 1]
```diff
@@ -217,11 +217,18 @@ export function pickerOptionsFromAccept(accept) {
 
         return ok;
       })
-      .map(([mimeType, ext]) => ({
-        accept: {
+      .reduce(
+        (agg, [mimeType, ext]) => ({
```
Comment (diogoredin) [inline src/utils/index.js:221]: args?

**Review by diogoredin — APPROVED** [review]
LGTM 🚀

File: src/utils/index.js (hunk 1 above)
Comment (unimonkiez) [reply src/utils/index.js:221]: Short for aggregate, usually the name I give to the first argument of reduce function.  
Want me to change it to something else?

Comment (paulhklam1122) [reply src/utils/index.js:221]: Makes sense, please merge ASAP. Would be greatly appreciated.

**Comment by taylorhakes** [comment]
@rolandjitsu Are you against this pull request and you won’t merge it? Would you prefer we fork this repo? Wondering because it seems like a ton of people are waiting on this fix

**Comment by taylor-cedar** [comment]
@rolandjitsu  Are you against this pull request and you won’t merge it? Would you prefer we fork this repo? Wondering because it seems like a ton of people are waiting on this fix

**Comment by rolandjitsu** [comment]
Sorry folks, I was on holiday. I'll have another look soon.

**Comment by fatihcandev** [comment]
When do you think this would be merged and a new version would be released? Currently working on a new feature that depends on react-dropzone.

**Review by adri — APPROVED** [review]

**Review by abraganu — APPROVED** [review]

**Comment by lucamicheli** [comment]
Please merge this PR to fix this issue. We are relying on drop zone and is causing issues to our customers. Thank you

**Review by LisaFelder — APPROVED** [review]

**Comment by omerman** [comment]
Please merge this pr.

**Review by rolandjitsu — APPROVED** [review]

**Pull request merged**